int32 scanId
---
string response_type
string message

